src/components/LHNOptionsList/OptionRowLHNDataWithFocus.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import withCurrentReportID, {withCurrentReportIDPropTypes, withCurrentReportIDDefaultProps} from '../withCurrentReportID';
+import OptionRowLHNData from './OptionRowLHNData';
+
+const propTypes = {
+    ...withCurrentReportIDPropTypes,
+    shouldDisableFocusOptions: PropTypes.bool,
+};
+
+const defaultProps = {
+    ...withCurrentReportIDDefaultProps,
+    shouldDisableFocusOptions: false,
+};
+
+function OptionRowLHNDataWithFocus({currentReportID, shouldDisableFocusOptions, ...props}) {
```

Comment:
Can we add some comments stating what this `OptionRowLHNDataWithFocus ` does? It would be cool in future to have an understanding of what it does because not everyone will be coming back to the PR to understand.

Reply:
Done ✅